### PR TITLE
Chinese localization

### PR DESCRIPTION
**Pull request by xianli123**
As a Chinese native speaker, I reviewed all Chinese localization files and optimized the localization wordings. This branch includes the original localization and some updates.

### CODE REVIEW

**Comment by xianli123** [comment]
Hi @ssilvert @jonkoops I created a new Chinese Localization pr here. The main files have been reviewed by Haley and I updated them. Please take a look and merge it. Thanks!

File: js/apps/admin-ui/public/locales/ca/common-help.json [hunk 1]
```diff
@@ -1 +1,7 @@
-{}
\ No newline at end of file
+{
```
Comment (jonkoops) [inline js/apps/admin-ui/public/locales/ca/common-help.json:1]: This is somehow in the `ca` directory, I don't think that is intentional?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/public/locales/ca/common-help.json (hunk 1 above)
Comment (xianli123) [reply js/apps/admin-ui/public/locales/ca/common-help.json:1]: @jonkoops I didn't edit this file. I am not sure what this file is. Can we remove it?

File: js/apps/admin-ui/public/locales/ca/common-help.json [hunk 2]
```diff
@@ -1 +1,7 @@
...
+  "helpToggleInfo": "此开关将启用/禁用UI中的部分帮助信息。包括任何帮助文本、链接和弹出窗口",
+  "showPassword": "以明文展示密码字段",
+  "helpFileUpload": "上传 JSON 文件",
+  "helpFileUploadClient": "上传 JSON 或 XML 文件",
+  "dragHelp": "按空格键或回车键开始拖动，并使用箭头键向上或向下导航。按回车键确认拖动，或按任何其他键取消拖动操作"
```
Comment (xianli123) [inline js/apps/admin-ui/public/locales/ca/common-help.json:6]:
```suggestion

```

File: js/apps/admin-ui/public/locales/ca/common-help.json [hunk 3]
```diff
@@ -1 +1,2 @@
-{}
\ No newline at end of file
+{
```
Comment (xianli123) [inline js/apps/admin-ui/public/locales/ca/common-help.json:1]:
```suggestion
```

**Comment by jonkoops** [comment]
Superseded by #20752

**Pull request closed without merging**